app/src/main/java/org/oppia/app/onboarding/OnboardingFragmentPresenter.kt

```diff
@@ -45,16 +45,22 @@ class OnboardingFragmentPresenter @Inject constructor(
   private fun setUpViewPager() {
     onboardingPagerAdapter = OnboardingPagerAdapter(fragment.requireContext())
     binding.onboardingSlideViewPager.adapter = onboardingPagerAdapter
-    binding.onboardingSlideViewPager.addOnPageChangeListener(object : ViewPager.OnPageChangeListener {
+    binding.onboardingSlideViewPager.addOnPageChangeListener(object :
+      ViewPager.OnPageChangeListener {
       override fun onPageScrollStateChanged(state: Int) {
+        val position: Int = binding.onboardingSlideViewPager.currentItem
+        if (state.equals(1) && binding.onboardingSlideViewPager.currentItem > 0)
```

Comment:
Nit:
`if (state == 1 && binding.onboardingSlideViewPager.currentItem > 0)`

Reply:
+1

Reply:
changed to if (state == 1 && binding.onboardingSlideViewPager.currentItem > 0)